### ISSUE TITLE
Ability to hide side menu in explorer

### ISSUE THREAD

**Issue by JustLuuuu** [kodadot/nft-gallery#5489]
Check the video that will give you a better idea. After scrolling down in explorer, I cannot hide the side menu. I would like to have the option to hide/unhide it even after scrolling without the need to go back on top. 

https://user-images.githubusercontent.com/104380776/229290940-9d9b2a75-7c77-456f-8908-a208bfc414b1.mov

**Comment by exezbcz** [comment]
we would have to then stick the whole strip to the top where the tab switch, sort by etc. is
![image](https://user-images.githubusercontent.com/90852205/229451220-f7fd5bcf-4c57-466b-8dd8-048c06b5f8f6.png)
- less vertically space and it looks ugly
- rarible does not have it either


I think its better than having double navbar on top, see opensea
![image](https://user-images.githubusercontent.com/90852205/229451636-bb0f9b31-dd12-43a9-b388-8598cfa15dd0.png)

**Comment by JustLuuuu** [comment]
Hmm.. that's true, and I agree with that. But I thought about it, and the best solution would be something like this: What if we added a small double arrow button at the bottom? Something like this:
<img width="1443" alt="Screenshot 2023-04-03 at 10 23 14" src="https://user-images.githubusercontent.com/104380776/229453470-1df25647-99a0-40d8-b9ec-322cd0102970.png">

It would not be very disruptive to design and it will give us extra functionality. I think people are used to double arrows from online shopping and similar pages.

**Comment by exezbcz** [comment]
@JustLuuuu yup, I will think about it. Mainly about the case when the content fills out the whole burger menu

**Comment by roiLeo** [comment]
Or we could reintroduce "scroll to top" btn

**Comment by exezbcz** [comment]
@roiLeo Indeed, I noticed it was missing while testing:D

I will create issue, we can leave this one as a backlog since the scroll up does not fully solve the problem.

**Comment by VladoTheBoi** [comment]
Nifty has it like this which seems like a good solution. 
<img width="1210" alt="Screenshot 2023-04-03 at 12 13 43" src="https://user-images.githubusercontent.com/95911767/229481109-12719a29-65b2-4165-bbeb-d6e0ec87219f.png">
<img width="1205" alt="Screenshot 2023-04-03 at 12 13 50" src="https://user-images.githubusercontent.com/95911767/229481131-d084c17f-3fd7-4bf7-af85-12ccbcff4512.png">
